[VI] Add `Rules/Code of conduct for modding and mapping`

summyvn opened the pull request:
<!-- 
  - Use [x] to complete the items
  - Remove the items unrelated to your work
  - Add any relevant information you consider useful
  - If there are no reviewers for your language, please mention it explicitly
-->

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [ ] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

vietng322611 commented on wiki/Rules/Code_of_conduct_for_modding_and_mapping/vi.md:
```diff
@@ -0,0 +1,60 @@
+# Quy tắc ứng xử với modding và mapping cho cộng tác viên
+
+**Quy tắc ứng xử đối với modding và mapping** là quy tắc và cũng là hướng dẫn áp dụng cho toàn bộ cộng đồng [modding](/wiki/Modding) và [mapping](/wiki/Beatmapping) cũng như phần mở rộng cho [Quy tắc cộng đồng osu!](/wiki/Rules) và [Tiêu chí xếp hạng chung](/wiki/Ranking_criteria). Để đảm bảo rằng các cuộc thảo luận mapping và modding ở các chủ đề diễn ra trong môi trường mang tính xây dựng, tích ...
```
**Quy tắc ứng xử đối với modding và mapping** là những quy tắc, hướng dẫn được áp dụng cho toàn bộ cộng đồng [modding](/wiki/Modding) và [mapping](/wiki/Beatmapping), cũng như là phần mở rộng cho [Quy tắc cộng đồng osu!](/wiki/Rules) và [Tiêu chí xếp hạng chung](/wiki/Ranking_criteria). Để đảm bảo rằng các cuộc thảo luận về mapping và modding ở các chủ đề diễn ra trong môi trường mang tính xây dựng, tích cực và hiệu quả, các quy tắc ứng xử là rất quan trọng để mọi người ở cùng một chủ đề. Khi tham gia vào cộng đồng modding và mapping, bắt buộc phải tuân theo các quy tắc ứng xử này. Hành vi vi phạm các quy tắc này có thể dẫn đến [hình phạt đối với tài khoản của bạn](/wiki/Silence).

vietng322611 commented on wiki/Rules/Code_of_conduct_for_modding_and_mapping/vi.md:
```diff
@@ -0,0 +1,60 @@
+
+## Hành vi và ứng xử cần làm
+
+- Mục đích của modding là giúp người lập bản đồ cải thiện công việc giúp beatmap của họ lên trạng thái [xếp hạng](/wiki/Beatmap/Category#ranked) nếu người lập bản đồ chọn làm như vậy. Nếu bạn không thích việc giúp đỡ người khác, xin hãy tránh xa các cuộc thảo luận để tránh *làm tổn thương họ*.
```
- Mục đích của modding là giúp người tạo beatmap cải thiện beatmap của họ để có thể đưa lên trạng thái [xếp hạng](/wiki/Beatmap/Category#ranked) (nếu muốn). Nếu bạn không muốn giúp đỡ người khác, xin hãy tránh xa các cuộc thảo luận để tránh *làm tổn thương họ*.

vietng322611 commented on wiki/Rules/Code_of_conduct_for_modding_and_mapping/vi.md:
```diff
@@ -0,0 +1,60 @@
+## Hành vi và ứng xử cần làm
+
+- Mục đích của modding là giúp người lập bản đồ cải thiện công việc giúp beatmap của họ lên trạng thái [xếp hạng](/wiki/Beatmap/Category#ranked) nếu người lập bản đồ chọn làm như vậy. Nếu bạn không thích việc giúp đỡ người khác, xin hãy tránh xa các cuộc thảo luận để tránh *làm tổn thương họ*.
+  - Đừng né tránh những người lập bản đồ và người sửa đổi mới — những người này là những người cần nhiều kiến thức nhất từ ​​ý kiến ​​đóng góp của bạn và mọi người! Hãy kiên nhẫn, thấu hiểu và đưa ra càng nhiều lời giải thích càng tốt để họ có thể học hỏi và tiến bộ nhanh hơn.
```
- Đừng né tránh những người mới tham gia mapping và modding  — họ rất cần những ​​ý kiến ​​đóng góp của bạn và mọi người! Hãy kiên nhẫn, thấu hiểu và đưa ra càng nhiều lời giải thích càng tốt để họ có thể học hỏi và tiến bộ nhanh hơn.

vietng322611 commented on wiki/Rules/Code_of_conduct_for_modding_and_mapping/vi.md:
```diff
@@ -0,0 +1,60 @@
+- Mục đích của modding là giúp người lập bản đồ cải thiện công việc giúp beatmap của họ lên trạng thái [xếp hạng](/wiki/Beatmap/Category#ranked) nếu người lập bản đồ chọn làm như vậy. Nếu bạn không thích việc giúp đỡ người khác, xin hãy tránh xa các cuộc thảo luận để tránh *làm tổn thương họ*.
+  - Đừng né tránh những người lập bản đồ và người sửa đổi mới — những người này là những người cần nhiều kiến thức nhất từ ​​ý kiến ​​đóng góp của bạn và mọi người! Hãy kiên nhẫn, thấu hiểu và đưa ra càng nhiều lời giải thích càng tốt để họ có thể học hỏi và tiến bộ nhanh hơn.
+  - Nếu bạn là người mới, đừng ngại khi nói vậy! Điều này sẽ thu hút các thành viên cộng đồng có kinh nghiệm chỉ cho bạn đi đúng hướng. Mọi người đều mới và ở vị trí của bạn một lần!
+- Hãy đối xử với người khác như cách bạn muốn được đối xử. Mọi người dễ tiếp thu những lời đề nghị tử tế, hữu ích hơn những lời tục tĩu. Cố gắng hiểu quan điểm của người khác thay vì tấn công nó - không ai thích cảm thấy mình vô dụng, kém cỏi hay ngu ngốc. Nếu bạn là một kẻ độc hại, hãy cho rằng tất cả những người khác đều không giống bạn.
```
- Hãy đối xử với người khác như cách bạn muốn được đối xử. Mọi người dễ tiếp thu những lời đề nghị tử tế, hữu ích hơn những lời tục tĩu. Cố gắng hiểu quan điểm của người khác thay vì tấn công nó - không ai thích cảm thấy mình vô dụng, kém cỏi hay ngu ngốc. Nếu bạn thích bị công kích, nhớ rằng không phải ai cũng giống bạn.

vietng322611 commented on wiki/Rules/Code_of_conduct_for_modding_and_mapping/vi.md:
```diff
@@ -0,0 +1,60 @@
+
+## Tạo một bài đăng mod
+
+Nếu bạn tải xuống một bản đồ và tìm ra một số điều khiến bạn không hài lòng — giờ là lúc bạn bày tỏ suy nghĩ của mình và giúp cải thiện bản đồ. Nhưng làm cách nào bạn có thể khiến người lập bản đồ hiểu được quan điểm của mình?, xem đề xuất của bạn và đưa ra phản hồi cho bạn? Suy cho cùng, không ai thích cảm giác rằng thời gian của mình bị lãng phí.
```
Bạn tải xuống một beatmap và tìm ra một số điều khiến bạn không hài lòng — đó là lúc bạn bày tỏ suy nghĩ của mình và giúp cải thiện nó. Nhưng làm cách nào để người tạo beatmap hiểu, thử và đưa ra phản hồi cho các đề xuất của bạn? Suy cho cùng, không ai thích cảm giác rằng thời gian của mình bị lãng phí.

vietng322611 reviewed (changes requested):
> Beatmap/map cannot be translated as "bản đồ", it's a term in osu, not an actual word in English. This make your translation look like it came from google translator.
> Many words/phrases are wrongly translated, I have fixed some but there still a lot of them.
> And please don't use google translator, that's the last resort. Use the word in English directly if there are no alternatives.

summyvn commented:
> * Beatmap/map cannot be translated as "bản đồ", it's a term in osu, not an actual word in English. This make your translation look like it came from google translator.
> * Many words/phrases are wrongly translated, I have fixed some but there still a lot of them.
> * And please don't use google translator, that's the last resort. Use the word in English directly if there are no alternatives.

I fixed it
